> test/sanity/validate-modules/ignore.txt:57:1: A102 Remove since "lib/ansible/modules/cloud/amazon/ec2_eni.py" passes "E324" test

Maybe I'm doing something wrong, but running that same `ansible-test sanity --test validate-modules` on my branch yields almost 400 errors:

> ERROR: Found 395 validate-modules issue(s) which need to be resolved:

Don't get me wrong: I am glad to apply the fix ansibot suggested, but is that the _expected_ output from `ansible-test`?

Or, worse, why was E324 suppressed in the first place? It feels like if the one item that I fixed broke a suppression, but that item negatively affected an actual user, then the value of the suppression is suspicious.

--- a/lib/ansible/modules/cloud/amazon/ec2_eni.py
+++ b/lib/ansible/modules/cloud/amazon/ec2_eni.py
@@ -69,16 +69,19 @@ options:
       - Force detachment of the interface. This applies either when explicitly detaching the interface by setting instance_id
         to None or when deleting an interface with state=absent.
     default: 'no'
+    type: bool
   delete_on_termination:
     description:
       - Delete the interface when the instance it is attached to is terminated. You can only specify this flag when the
         interface is being modified, not on creation.
     required: false
+    type: bool
   source_dest_check:
     description:
       - By default, interfaces perform source/destination checks. NAT instances however need this check to be disabled.
         You can only specify this flag when the interface is being modified, not on creation.
     required: false
+    type: bool
   secondary_private_ip_addresses:
     description:
       - A list of IP addresses to assign as secondary IP addresses to the network interface.
@@ -90,6 +93,7 @@ options:
       - To be used with I(secondary_private_ip_addresses) to determine whether or not to remove any secondary IP addresses other than those specified.
         Set secondary_private_ip_addresses to an empty list to purge all secondary addresses.
     default: no
+    type: bool
     version_added: 2.5
   secondary_private_ip_address_count:
     description:
@@ -102,6 +106,7 @@ options:
         to be reassigned to the specified network interface.
     required: false
     default: 'no'
+    type: bool
     version_added: 2.7
 extends_documentation_fragment:
     - aws

--- a/test/sanity/validate-modules/ignore.txt
+++ b/test/sanity/validate-modules/ignore.txt
@@ -54,7 +54,6 @@ lib/ansible/modules/cloud/amazon/ec2_elb.py E326
 lib/ansible/modules/cloud/amazon/ec2_elb_facts.py E323
 lib/ansible/modules/cloud/amazon/ec2_elb_lb.py E324
 lib/ansible/modules/cloud/amazon/ec2_elb_lb.py E325
-lib/ansible/modules/cloud/amazon/ec2_eni.py E325
 lib/ansible/modules/cloud/amazon/ec2_group.py E322
 lib/ansible/modules/cloud/amazon/ec2_group.py E325
 lib/ansible/modules/cloud/amazon/ec2_instance.py E324

